client/src/modules/stock/inventories/templates/inventory.action.html

```diff
@@ -42,13 +42,16 @@
     <li class="divider"></li>
 
     <li>
-      <a data-method="view-amc-calculations" href ng-click="grid.appScope.viewAMCCalculations(row.entity)">
+      <a data-method="view-amc-calculations" href
+        ng-click="grid.appScope.viewAMCCalculations(row.entity)">
         <i class="fa fa-newspaper-o"></i> <span translate>INVENTORY.OPEN_AMC_CALCULATION</span>
       </a>
     </li>
 
     <li>
-      <a ng-href="/reports/stock/sheet?{{ grid.appScope.openStockSheetReport(row.entity) }}" target="_blank">
+      <a uib-tooltip="{{ 'REPORT.STOCK.INVENTORY_REPORT_TOOLTIP' | translate }}"
```

Comment:
Can you make this jut off the left side instead of putting it over the menu?  `uib-tooltip-placement="left"`

![image](https://user-images.githubusercontent.com/896472/110851154-f5863380-82b0-11eb-9d8e-4d0a7a0b24b1.png)

Reply:
Fixed

Reply:
The fix was:   tooltip-placement="left"  (not uib-tooltip-placement)
and it worked nicely.    I liked it so much I went back and tweaked the menu for auto merging lots in the Find Duplicate Lots page!